build.gradle.kts

```diff
@@ -1,16 +1,16 @@
 plugins {
     `java-gradle-plugin`
```

Comment:
can be removed I think, seems to be applied automatically since 1.0.0 (https://blog.gradle.org/simplifying-the-plugin-publish-plugin#stronger-conventions)

Reply:
ahh, thanks!